releases/release-4.0.6.md

```diff
@@ -0,0 +1,196 @@
+    - Fix read failure from the `tiflash_segments` and `tiflash_tables` system tables when some TiFlash nodes are offline [#19748](https://github.com/pingcap/tidb/pull/19748)
+    - Fix a wrong result of the `Count(col)` aggregation function [#19628](https://github.com/pingcap/tidb/pull/19628)
+    - Fix a runtime error of the `TRUNCATE` operation [#19445](https://github.com/pingcap/tidb/pull/19445)
+    - Fix the issue that `PREPARE statement FROM @Var` will fail when `Var` contains uppercase characters [#19378](https://github.com/pingcap/tidb/pull/19378)
+    - Fix a bug that schema charset modification in an uppercase schema will cause panic [#19302](https://github.com/pingcap/tidb/pull/19302)
+    - Fix inconsistency between `information_schema.statements_summary` and `explain` plan, when the information contains `tikv/tiflash` [#19159](https://github.com/pingcap/tidb/pull/19159)
+    - Fix the error in tests that the file does not exist for `select into outfile` [#19725](https://github.com/pingcap/tidb/pull/19725)
+    - Fix the issue that `INFORMATION_SCHEMA.CLUSTER_HARDWARE` does not have raid device information [#19457](https://github.com/pingcap/tidb/pull/19457)
+    - Make the `add index` operation that has a generated column with the `case-when` expression can exit normally when it encounters a parse error [#19395](https://github.com/pingcap/tidb/pull/19395)
+    - Fix a bug that DDL takes too long to retry [#19488](https://github.com/pingcap/tidb/pull/19488)
+    - Make statements like `alter table db.t1 add constraint fk foreign key (c2) references t2(c1)` execute without first executing `use db` [#19471](https://github.com/pingcap/tidb/pull/19471)
+    - Change the dispatch error from an Error to an Info message in the server log file [#19454](https://github.com/pingcap/tidb/pull/19454)
+
++ TiKV
+
+    - Fix the estimation error for a non-index column when collation is enabled [#8620](https://github.com/tikv/tikv/pull/8620)
+    - Fix the issue that green GC might miss locks during Region transfer [#8460](https://github.com/tikv/tikv/pull/8460)
+    - Fix the CDC incorrect resolved TS timeout [#8573](https://github.com/tikv/tikv/pull/8573)
+    - Fix a panic issue if a TiKV runs very slow during conf change [#8497](https://github.com/tikv/tikv/pull/8497)
+    - Fix the deadlock issue between the PD client thread and other threads calling PD sync requests [#8612](https://github.com/tikv/tikv/pull/8612)
+    - Update jemalloc to 5.2.1 to address memory allocation problem in huge page [#8463](https://github.com/tikv/tikv/pull/8463)
+    - Fix unified thread pool hang for long running query. [#8427](https://github.com/tikv/tikv/pull/8427)
+
++ PD
+
+    - Add the `initial-cluster-token` configuration to prevent different clusters from communicating to each other during bootstrap [#2922](https://github.com/pingcap/pd/pull/2922)
```

Comment:
```suggestion
    - Add the `initial-cluster-token` configuration to prevent different clusters from communicating with each other during bootstrap [#2922](https://github.com/pingcap/pd/pull/2922)
```